apps/src/gamelab/animationListModule.js

```diff
@@ -321,17 +385,38 @@ export function addAnimation(key, props) {
 export function addLibraryAnimation(props) {
   return (dispatch, getState) => {
     const key = createUuid();
-    dispatch({
-      type: ADD_ANIMATION,
-      key,
-      props
-    });
+    dispatch(addAnimationAction(key, props));
     dispatch(loadAnimationFromSource(key, () => {
       dispatch(selectAnimation(key));
     }));
     let name = generateAnimationName(props.name, getState().animationList.propsByKey);
     dispatch(setAnimationName(key, name));
-    dashboard.project.projectChanged();
+    projectChanged();
+  };
+}
+
+/**
+ * After pending frames are added, remove them as pending.
+ * @returns {function}
+ */
+export function removePendingFrames() {
+  return dispatch => {
+    dispatch(removePendingFramesAction());
+  };
+}
```

Comment:
This method is unnecessary. We can just export `removePendingFramesAction` and dispatch that directly from our component.